### ISSUE TITLE
Request : continuous document display

### ISSUE THREAD

**Issue by ImaCrea** [ONLYOFFICE/DocumentServer#1446]
Hello, I've been searching in display options and in issues here and couldn't find something about such topic.

I was wondering if there's a way to have a kind of "continuous" mode of display for document _àla_ google doc or hedgedoc/pad.

Most of the documents I work on are not meant to be printed, so I don't really need to have it displayed through separated pages. In LibreOffice it's done by switching between "normal" and "web" mode in the "view" panel.

Thanks :)

**Comment by ShockwaveNN** [comment]
Hi, thanks for your idea

We already thought about this - there is issue 48757 in our private issue tracker

But seems it's a little bit complicated to implement, so currently there is no advancements on this task

**Comment by Eric1212** [comment]
I prefer the page-per-page in my context but an option to see it in a continuous way is something i like to see even if it's would not be my default settings.

**Comment by marksjus** [comment]
Agreed, I really like something like "Immersive reader" from MS Word. It's perfect for proofreading, when you're not concerned about the layout and you can focus on text.
